build.html

```diff
@@ -15,41 +15,40 @@ <h2 class="aligncenter">Authentication</h2>
         <p class="email-error text-danger">\{{ securityError }}</p>
       </template>
       <button class="btn btn-primary verify-identity" v-bind:class="{ disabled: disableButton }" type="submit">\{{ sendValidationLabel }}</button>
-      <button v-if="storedEmail" v-on:click="showVerify" class="btn btn-link have-code">I have an authentication code</button>
+      <button v-if="storedEmail" v-on:click="showVerify" class="btn btn-link have-code">I have an verification code</button>
     </form>
   </div>
 
   <div class="state" v-bind:class="{ past: confirmation, present: verifyCode, future: loading || auth, 'new-pin': resentCode }" data-state="verify-code">
     <form v-on:submit.prevent="validate">
       <h2>Verify your email address</h2>
-      <p>Enter the authentication code that was sent to the phone number associated with <span class="input-email">\{{ storedEmail }}</span>.</p>
+      <p>Enter the verification code that was sent to the phone number associated with <span class="input-email">\{{ storedEmail }}</span>.</p>
       <div class="form-group input-wrapper" v-bind:class="{ 'has-error': codeError}">
-        <input type="text" v-model="code" class="verify_pin form-control" placeholder="Enter authentication code">
+        <input type="text" v-model="code" class="verify_pin form-control" placeholder="Enter verifiation code">
         <span v-on:click="back" class="fa fa-chevron-left back"></span>
       </div>
-      <p class="pin-error text-danger">You entered the wrong authentication code or it might have expired.
-        <br>Try again or tap <i>Resend authentication code</i>.</p>
+      <p class="pin-error text-danger">You entered the wrong verification code or it might have expired.
+        <br>Try again or tap <i>Resend verification code</i>.</p>
       <button class="btn btn-primary verify-code" type="submit">Verify</button>
-      <button v-on:click="resendCode" class="btn btn-link resend-code">Resend authentication code</button>
-      <p class="pin-resent text-success">A new authentication code was sent to your email.</p>
+      <button v-on:click="resendCode" class="btn btn-link resend-code">Resend verification code</button>
+      <p class="pin-resent text-success">A new verification code was sent to your email.</p>
     </form>
   </div>
 
   <div class="state" v-bind:class="{ past: verifyCode, present: confirmation, future: loading || auth || verifyCode }" data-state="confirmation">
-    <h2>Welcome</h2>
-    <p>You have been authenticated and you can now continue to the app.</p>
+    <p>You have been verified and you can now continue.</p>
```

Comment:
Sorry, I meant this part - I think you used `proceed` in the other one. Just check every line carefully and confirm they're the same.